kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -1171,6 +1171,24 @@ object KyuubiConf {
       .booleanConf
       .createWithDefault(false)
 
+  val KUBERNETES_INFORMER_PERIOD: ConfigEntry[Long] =
+    buildConf("kyuubi.kubernetes.informer.period")
+      .doc("Kubernetes Informer poll driver pod period." +
+        "Set too small can lead to, stress on Kubernetes Api Server; " +
+        "Set too lager can lead to, app info can't updated in time ")
+      .version("1.8.0")
+      .timeConf
+      .checkValue(_ > 0, "must be positive number")
```

Comment:
From the JavaDoc of `io.fabric8.kubernetes.client.dsl.Informable#runnableInformer`

> the resync period or 0 for no resync